src/App.tsx

```diff
@@ -3,11 +3,17 @@ import styled from 'styled-components';
 import SummaryCard from './components/SummaryCard';
 
 import background from './assets/pattern-background-desktop.svg';
+import hero from './assets/illustration-hero.svg';
 
 const App = () => {
```

Comment:
The state for the selected PricePlan should be here

Reply:
I'm not too sure what to do here, as this is one of the original branches and the logic will change quite a lot in MEN-5 branch which is to do with the creation of the PricePlan component itself. 
What I have done is simply added a `pricePlanObj` to `App.tsx` and have passed this as a prop to the `PriceCard` component.

Reply:
In this case, we are introducing a dependency between the 2 branches. We will have to wait for that branch to get fixed. Before we can progress on this one.